integration_tests/ipp/container/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM zgrab2_service_base:latest
+
+RUN apt-get update
+RUN apt-get install -y cups
+RUN apt-get install -y cups-pdf
+RUN echo "ServerAlias *" >> /etc/cups/cupsd.conf
+RUN echo "FileDevice yes" >> /etc/cups/cupsd.conf
+RUN sed -i 's/Order allow,deny/Order allow,deny\'$'\n  Allow all/g' /etc/cups/cupsd.conf
+RUN sed -i 's/Listen localhost:631/Listen 0.0.0.0:631/g' /etc/cups/cupsd.conf
```

Comment:
Using `sed` replacements ties this to a specific version of cups (and/or its packaging); this is fine for now, but maybe add a TODO comment to provide our own cupsd.conf rather than modifying the default.